deploy.sh

```diff
@@ -0,0 +1,38 @@
+# !/bin/bash
```

Comment:
Wouldn't it be possible to simplify the process following [this](https://docs.travis-ci.com/user/deployment/pages)?

Reply:
Yes. I haven't seen this before